PhysicsTools/PatAlgos/python/slimming/slimmedPhotons_cfi.py

```diff
@@ -11,7 +11,7 @@
     linkToPackedPFCandidates = cms.bool(True),
     recoToPFMap = cms.InputTag("reducedEgamma","reducedPhotonPfCandMap"),
     packedPFCandidates = cms.InputTag("packedPFCandidates"),
-    saveNonZSClusterShapes = cms.string("(r9()>0.8 || chargedHadronIso()<20 || chargedHadronIso()<0.3*pt())"), # save additional user floats: (sigmaIetaIeta,sigmaIphiIphi,sigmaIetaIphi,r9,e1x5_over_e5x5)_NoZS 
+    saveNonZSClusterShapes = cms.string("0"), #(r9()>0.8 || chargedHadronIso()<20 || chargedHadronIso()<0.3*pt())"), # save additional user floats: (sigmaIetaIeta,sigmaIphiIphi,sigmaIetaIphi,r9,e1x5_over_e5x5)_NoZS 
```

Comment:
Those are those extra userFloats, right? I agree to drop them...
I know some analysts still use them, but it's more than time to use the class member directly...